hypha/apply/funds/templates/funds/grouped_application_list.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base-apply.html" %}
+{% load static %}
+{% load render_bundle from webpack_loader %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'css/apply/fancybox.css' %}">
+{{ filter.form.media.css }}
+{% endblock %}
+
+{% block content%}
+<div class="admin-bar">
+        <div class="admin-bar__inner wrapper--search">
+                {% block page_header %}
+                    <div>
+                        <h1 class="gamma heading heading--no-margin heading--bold">List of Applications</h1>
```

Comment:
Add summary keyword so it can be "Submissions summary". Submissions keyword is used on other pages header so use it instead if Applications.